components/brave_wallet/browser/zcash/zcash_create_transparent_to_orchard_transaction_task.h

```diff
@@ -58,12 +56,13 @@ class ZCashCreateShieldTransactionTask {
   bool CreateTransaction();
 
   const raw_ref<ZCashWalletService> zcash_wallet_service_;  // Owns `this`.
-  std::string chain_id_;
-  mojom::AccountIdPtr account_id_;
+  ZCashActionContext context_;
   OrchardAddrRawPart receiver_;
   std::optional<OrchardMemo> memo_;
   uint64_t amount_;
 
+  bool started_ = false;
```

Comment:
Why we need these `started_` flags for tasks?

Reply:
Just to ensure that Start is called only once